parmed/dlpoly/__init__.py

```diff
@@ -0,0 +1,46 @@
+"""
+Contains classes for parsing DLPOLY topology and parameter files
+"""
+import os as _os
+from parmed.utils import which as _which
```

Comment:
hi, what's the aim of this?

Reply:
I implemented the functionalities on this PR starting from what was already in ParmEd for Gromacs. I am not sure why these `as` choices were employed.

Reply:
I see.

Reply:
It prevents them from being imported with `from parmed.dlpoly import *`